ovgs.proto

```diff
@@ -0,0 +1,365 @@
+// Copyright (c) 2023 Arista Networks, Inc.
+// Use of this source code is governed by the BSD 2-Clause License
```

Comment:
This should reference Apache 2.0 license

Reply:
ack - updated.